DependencyInjection/Compiler/TwigExceptionPass.php

```diff
@@ -26,5 +26,9 @@ public function process(ContainerBuilder $container)
         if ($container->has('fos_rest.exception_listener') && $container->has('twig.exception_listener')) {
             $container->removeDefinition('twig.exception_listener');
         }
+
+        if (!$container->has('templating.engine.twig')) {
+            $container->removeDefinition('fos_rest.exception.twig_controller');
```

Comment:
actually I think the "trick" with making the service private is likely causing a problem:
https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1288#issuecomment-167811904